.gitlab-ci.yml

```diff
@@ -566,7 +609,7 @@ scenario_tests_mac_python27_10_13:
   script:
     - export VIRTUALENV="/Library/Frameworks/Python.framework/Versions/2.7/bin/virtualenv"
     - cd scenario-tests
-    - ./run_scenario_tests.sh ../target/turicreate-*.whl
+    - ./run_scenario_tests.sh "$(ls ../target/turicreate-*.whl | grep -v minimal)"
```

Comment:
Again, this can be replaced with `./run_scenario_tests.sh ../target/turicreate-*minimal.whl`